docusaurus/docs/iOS/03-guides/14-swiftui-vs-uikit.mdx

```diff
@@ -26,11 +26,10 @@ struct CallView: View {
     }
 
     var body: some View {
-        HomeView(viewModel: viewModel)
+        Color.clear // You can use any of your views.
```

Comment:
Thinking if this can be confusing from customers perspective? Maybe create a simple `HomeView` that has a button or something?

Reply:
I made this change to avoid defining the HomeView in the test project but i ended up defining it. So, I reverted the snippet's code.